cli/src/main.rs

```diff
@@ -12,73 +15,44 @@ struct Args {
     cmd: Commands,
 }
 
-#[derive(ValueEnum, Debug, Clone)]
-enum GameType {
-    Holdem,
-    Omaha,
-    Hilo,
-}
-
-#[derive(Subcommand, Debug, Clone)]
+#[derive(Parser, Debug, Clone)]
 enum Commands {
-    Init {
-        #[arg(short, long, value_enum, default_value = "holdem")]
-        game_type: GameType,
-
-        #[arg(short, long, num_args = NO_OF_PLAYERS as usize)]
-        player: Option<Vec<u64>>,
-        
-        #[arg(short, long, default_value_t = SMALL_BLIND_AMOUNT)]
-        small_blind: u8,
-
-        #[arg(short, long, default_value_t = BUY_IN_AMOUNT)]
-        buy_in: u64,
-
-        #[arg(short, long, value_parser)]
-        config: Option<std::path::PathBuf>,
-    },
-    Register {
-        #[arg(short, long)]
-        identifier: String,
-    }
+    Action(ActionCmd),
+    ConsumeNotes(ConsumeNotesCmd),
```

Comment:
Does each player need to run this command. to start consuming notes ?

Reply:
Yes for now, later maybe lets club this into aze-cli join, wdyt?